crates/runtime/src/lib.rs

```diff
@@ -65,6 +69,27 @@ impl Runtime {
             },
         }
     }
+
+    pub fn start_pods_watcher(&self) -> notify::Result<()> {
+        let mut current_app = self.app.clone();
+
+        let handle_event = move |event: podswatcher::PodsWatcherEvent| {
+            match event {
+                podswatcher::PodsWatcherEvent::PodsUpdated(new_app) => {
+                    tracing::info!("updated pods information: {:?}", new_app);
+                    tracing::info!("previous pods information: {:?}", current_app);
```

Comment:
Info level logging is visible to the customers - let's have a high bar for what we log here. This seems like it should be at a debug or trace level, and more concise update for the info level.

Reply:
Fixed, thank you!